.github/workflows/CI.yml

```
@@ -0,0 +1,120 @@
+# Run CI tests
+name: CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: main
+  pull_request:
+    branches: main
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # JOB to run change in the build files
+  changes:
+    runs-on: ubuntu-latest
+    # Required permissions
+    permissions:
+      pull-requests: read
+    # Set job outputs to values from filter step
+    outputs:
+      files: ${{ steps.filter.outputs.files }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.1.7
+
+      - name: Filter files
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 #v3.0.2
+        id: filter
+        with:
+          filters: |
+            files:
+              - 'pyproject.toml'
+              - 'setup.py'
+              - '.conda/env_build.yml'
+              - '.conda/meta.yml'
+
+  verify-conda-build:
+    name: Conda Build
+    runs-on: ubuntu-latest
+    needs: changes
+    # Only run if there are changes in the build files
+    if: ${{ needs.changes.outputs.files == 'true' }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup conda build environment
+        uses: conda-incubator/setup-miniconda@a4260408e20b96e80095f42ff7f1a15b27dd94ca # v3.0.4
+        with:
+          miniconda-version: "latest"
+          python-version: "3.10" # TODO: Add back in vars.PY_VERSION
+          environment-file: .conda/env_build.yml
+          auto-activate-base: false
+          auto-update-conda: false
+          show-channel-urls: true
+
+      - name: Verify conda recipe
+        shell: bash -el {0}
+        run: conda-verify .conda --ignore C2105,C2122
```

Just adding a couple comments to make the ignores more tractable. Why these?
```suggestion
        # Ignores: 
        # C2105 - Found invalid package version in meta.yaml
        # C2122 - Found invalid license family
        run: conda-verify .conda --ignore C2105,C2122
```

---

Thanks, I've added in that suggestion. I don't know much about the `conda-verify` tool honestly. I ran some tests with removing the `--ignore`, they added some log messages to the output, e.g. 
```
[C2105] Found invalid package version "0+untagged.1.gb1a7f19" in meta.yaml
[C2122] Found invalid license family "Apache"
```
But interestingly the check still passed, so I’ve added `--exit` flag so `conda-verify` exits with an error when one of the checks fails